### ISSUE TITLE
Rewrite tool examples in Markdown

### ISSUE THREAD

**Issue by ArionMiles** [iovisor/bpftrace#2880]
### Is your feature request related to a problem? Please describe.
Reading textfiles under `tools/` is an eysore. Most folks will read these fields by navigating from the root README.

### Describe the solution you'd like
I'd prefer to rewrite these files to markdown. It's rendered beautifully by the GitHub UI on the web and allows for richer formatting.

Since each text file is accompanied by the tool's source as well, I'd also consider re-organizing the tools dir to something like below:
```
├── bashreadline
│   ├── README.md
│   └── bashreadline.bt
```
where `README.md` has the contents of `bashreadline_example.txt` in markdown format.

This will enable users who click on the folder names to be presented with the markdown formatted example, and have the source present right there. GitHub renders any file named README.md insider a directory.

**Comment by ArionMiles** [comment]
I'd be happy to put in a PR for this

**Comment by ArionMiles** [comment]
I see the [CMakeLists.txt](https://github.com/iovisor/bpftrace/blob/master/tools/CMakeLists.txt) inside tools/ provides instructions for bundling the tool and the `*_examples.txt` files during compilation since man pages refer to these files.

I think rewriting & re-organizing the files may break this.

Assuming we do this, 
1. Can we still bundle the readme files and have them be as useful/readable as current text files?
2. Are these files in text format because they're intended to be read via commandline? Is breaking this functionality in favor of markdown acceptable?


I'm also curious about the syntax of this file, I don't know what it is.

**Comment by danobi** [comment]
Sorry about the radio silence :(

@jordalgo, remind me, wasn't there a desire to move the example text into the script itself?

**Comment by jordalgo** [comment]
> wasn't there a desire to move the example text into the script itself?

Yes, I believe so. I think that would be a nice improvement over having txt files.
